react/features/base/tracks/actions.js

```diff
@@ -717,3 +717,25 @@ export function updateLastTrackVideoMediaEvent(track, name) {
         name
     };
 }
+
+/**
+ * Toggles the facingMode constraint on the video stream.
+ *
+ * @returns {Function}
+ */
+export function toggleCamera() {
+    return async (dispatch, getState) => {
+        const state = getState();
+        const tracks = state['features/base/tracks'];
+        const localVideoTrack = getLocalVideoTrack(tracks).jitsiTrack;
+        const { _realDeviceId } = localVideoTrack;
+        const { videoInput } = state['features/base/devices'].availableDevices;
+        const { deviceId } = videoInput.find(camera => camera.deviceId !== _realDeviceId);
```

Comment:
We should get the other facing mode, here we are just getting another device.

Reply:
Not sure where I can retrieve the facing mode of the other camera without explicitly creating a track for it and thus having two tracks simultaneously, that's why I went with the `deviceId` approach. Any pointers would be appreciated :D

Reply:
You can use getSettings() on the existing track. I think JitsiLocalTravk may also have saved the facing mode.